docs/rl-algorithms/c51.md

```diff
@@ -0,0 +1,181 @@
+# Categorical DQN (C51)
+
+## Overview
+
+C51 introduces a distribution of perspective for DQN: instead of learning a single value for an action, C51 learns to predict a distribution of values for the action. With this perspective C51 introduces applies the Bellman's equation to learning the approximate value distribution. Empirically, C51 demonstrates impressive performance in ALE.
```

Comment:
- introduces a __distributional perspective__ for DQN ?

- With this perspective C51 introduces ... -> "To estimate the distribution of value for a given action,, Categorical DQN (C51) discretizes the support of the value distribution into N=51 atoms"

Reply:
I just simplified it...